onnx/defs/controlflow/defs.cc

```diff
@@ -734,15 +696,24 @@ ONNX_OPERATOR_SET_SCHEMA(
             AttributeProto::INT,
             true)
         .Attr(
-            "directions",
+            "scan_input_directions",
             "An optional list of M flags. The i-th element of the list specifies the direction "
             "to be scanned for the i-th scan_input tensor: 0 indicates forward direction and 1 "
             "indicates reverse direction. "
             "If omitted, all scan_input tensors will be scanned in the forward direction.",
             AttributeProto::INTS,
             false)
+        .Attr(
```

Comment:
axes attribute is missing.

How about we don't allow specifying an array, but an "I", which means callers want to scan [0, I]. Scan definition itself may enforce callers to do transpose outside if needed instead of specifying randomized axes and have Scan implementation to handle the complexity. What do you think?

Reply:
added axes attribute. I don't understand the second suggestion. (I understand eliminating axes attribute, but not the new attribute you are suggesting.)